pkg/actions/lua.go

```diff
@@ -128,6 +133,34 @@ func LuaRun(l *lua.State, code, name string) error {
 	return l.ProtectedCall(0, lua.MultipleReturns, 0)
 }
 
+func (h *LuaHook) collectStats(l *lua.State) {
+	// TODO(isan) how to configure?
+	packagesToReport := map[string]bool{
+		"lakefs/catalogexport/hive":             true,
+		"lakefs/catalogexport/symlink_exporter": true,
+		"lakefs/catalogexport/glue_exporter":    true,
+	}
```

Comment:
We like to collect information of require anything under lakefs.

Reply:
Good idea, done 👍